src/components/Admin/ManageLevels.js

```diff
@@ -0,0 +1,233 @@
+import React, { Component, Fragment } from "react";
```

Comment:
Decent amount of unused variables in here, use `eslint src/` to find them

Reply:
Resolved in commit c65a57b272e55c4fc0b26c5a3c8c836f7b3d43e1

Reply:
Resolved in commit 493b8b8e93bd0577dfe7df01ddb08fe50a4601df